adapters/algorix/algorixtest/params/race/banner.json

```diff
@@ -0,0 +1,4 @@
+{
+  "sid": "testSid",
```

Comment:
Prebid server core is not using these files to assert for shared memory race conditions. This file can be entirely removed.